draft-high-assurance-profile-oid4vc-sd-jwt-vc.md

```diff
@@ -176,40 +185,36 @@ All the requirements defined in [@!I-D.ietf-oauth-selective-disclosure-jwt] MUST
-* `type` JWT claim as defined in draft-terbu-sd-jwt-vc. For details on the `cnf` claim, see (#holder-key-resolution).
-* `status` claim MUST be present as defined in draft-terbu-sd-jwt-vc/draft-looker-jwt-cwt-status-list
-* For Cryptographic Holder Binding, HB-JWT MUST be supported as defined in [@!I-D.ietf-oauth-selective-disclosure-jwt].
+* It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT-VC. The wallet and the verifier  MUST support both mechanisms.
+* The `iss` claim MUST be an HTTPS URL. The `iss` value is used to obtain Issuer’s signing key as defined in (#issuer-key-resolution).
+* The `type` JWT claim as defined in [!I-D.ietf-terbu-sd-jwt-vc].
 
-Further claims, either registered or private JWT claims, can be added to the credential as required by the respective credential type, or determined by the Issuer or the Holder. All claims that are not understood by implementations MUST be ignored, as defined in section 4, [@!RFC7519].
-
-Note: If there is a requirement to communicate information about the verification status and identity assurance data of the claims about the subject, the syntax defined by @!OIDF.ekyc-ida SHOULD be used. It is up to each jurisdiction and ecosystem, whether to require it to the implementers of this profile.
+Note: Currently this profile only supports presentation of credentials with cryptographic Holder Binding: holder signature is required to bind presentation to a particular transaction and audience. This is planned to be expanded, once OpenID for Verifiable Credentials adds support for other forms of Holder Binding. See https://bitbucket.org/openid/connect/issues/1537/presenting-vc-without-a-vp-using-openid4vp
 
-Note: If there is a requirement to provide Subject’s identifier assigned and maintained by the Issuer, `sub` claim MAY be used. There is no requirement for a binding to exist between `sub` and `cnf` claims. See section X in @!I-D.draft-terbu-sd-jwt-vc for implementation considerations.
+* The public key passed in the `cnf` claim [@!RFC7800] MUST be a JSON Web Key [@!RFC7517] contained in the `jwk` claim. `kid' MUST be included in the JWK in the `cnf` claim, since it will be used in the JOSE Header.
 
-Note: Currently this profile only supports presentation of credentials with cryptographic Holder Binding: holder signature is required to bind presentation to a particular transaction and audience. This is planned to be expanded, once OpenID for Verifiable Credentials adds support for other forms of Holder Binding. See https://bitbucket.org/openid/connect/issues/1537/presenting-vc-without-a-vp-using-openid4vp
+Note: this requirement might change when claim-based and biometrics-based holder binding become available.
```

Comment:
this seems duplicative with line 192. how about the order of 194 -> 192&196 consolidated?

Reply:
192 is "It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT-VC. The wallet and the verifier  MUST support both mechanisms". I don't understand.

Reply:
the line numbers moved because of the changes accepted. with the new lines, it will be 198 -> 196&200 consoludated.

Reply:
reworked cnf/holder binding text